src/Playwright.Tests/BrowserContextRouteTests.cs

```diff
@@ -113,6 +113,38 @@ public async Task ShouldUnroute()
             Assert.AreEqual(new List<int>() { 1 }, intercepted);
         }
 
+        [PlaywrightTest("browsercontext-route.spec.ts", "should unroute page with base URL")]
```

Comment:
nit: as this test is specific to C#, you don't need to annotate it with the link to the upstream test.